pages.pt_BR/common/abduco.md

```diff
@@ -0,0 +1,24 @@
+# abduco
+
+> Gerenciador de sessao no terminal.
+> Mais informacoes: <http://www.brain-dump.org/projects/abduco/>.
```

Comment:
```suggestion
> Mais informações: <http://www.brain-dump.org/projects/abduco/>.
```

Reply:
I avoided using special characters to prevent strange messages in terminal. I'll fix it!